Simplified/Settings/NYPLSettingsAccountDetailViewController.m

```diff
@@ -323,6 +323,11 @@ - (void)setupViews
   [self setupTableData];
   
   self.syncSwitch = [[UISwitch alloc] initWithFrame:CGRectZero];
+  if (@available(iOS 12.0, *)) {
+    if (UIScreen.mainScreen.traitCollection.userInterfaceStyle == UIUserInterfaceStyleDark) {
+      [self.syncSwitch setOnTintColor:[NYPLConfiguration mainColor]];
```

Comment:
wondering if we should just set the onTintColor regardless of dark/light mode or OS...also I thought we were not supposed to use the library colors in Dark mode?

Reply:
Maybe I should use the `actionColor`, I will check with Marcus